packages/react-storage/src/components/StorageBrowser/controls/hooks/useStatusDisplay.ts

```diff
@@ -5,7 +5,7 @@ import { displayText } from '../../displayText/en';
 export const useStatusDisplay = (): StatusDisplayProps | null => {
   const { data, actionsConfig } = useControlsContext();
   const { taskCounts } = data;
-  const { isCancelable, type } = actionsConfig;
+  const { isCancelable, type } = actionsConfig ?? {};
```

Comment:
required because [actionsConfig is now optional](https://github.com/aws-amplify/amplify-ui/pull/5939/files#r1817235827)